src/issues.rs

```diff
@@ -92,15 +92,15 @@ impl Issues {
 
     /// returns a single page of issues results
     /// https://docs.atlassian.com/jira-software/REST/latest/#agile/1.0/board-getIssuesForBoard
-    pub fn list(&self, board: &Board, options: &SearchOptions) -> Result<IssueResults> {
+    pub fn list(&self, board: &Board, options: &SearchOptions) -> Result<Paginated<Issue>> {
```

Comment:
when I checked out master after merging this to I get the following compile issue because a breaking change with `IssueResults`. it was removed be replaced with `Paginated<Issue>` I took a closer look an noticed we weren't actually making use of the generalization. Im temporarily going to revert that part of this change until we actually need it.

```
   Compiling goji v0.2.4 (file:///Users/dougtangren/code/rust/goji)
error[E0412]: cannot find type `IssueResults` in this scope
  --> tests/issues_test.rs:16:18
   |
16 |     let results: IssueResults = serde_json::from_str(issue_results_str).unwrap();
   |                  ^^^^^^^^^^^^ not found in this scope

warning: unused import: `goji::issues::*`
 --> tests/issues_test.rs:4:5
  |
4 | use goji::issues::*;
```